Test/OrochiUtils.cpp

```diff
@@ -102,7 +66,7 @@ oroFunction OrochiUtils::getFunction( const char* code, const char* path, const
 	e = orortcCreateProgram( &prog, code, path, 0, 0, 0 );
 	std::vector<const char*> opts;
 	opts.push_back( "-I ../" );
-	opts.push_back( "-G" );
+	// opts.push_back( "-G" );
```

Comment:
This has to b removed in order to run on my machine. Otherwise there will be an error.
Need to check what this option does.